src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -227,7 +219,7 @@ public String getTabTitle() {
         if (getBibDatabaseContext().getDatabaseFile() == null) {
             title.append(GUIGlobals.UNTITLED_TITLE);
 
-            if (!database().getEntries().isEmpty()) {
+            if (!getDatabase().getEntries().isEmpty()) {
```

Comment:
I think the positive version `hasEntries` is more convenient and avoids the double negative `! hasNoEntries()`. If nobody veto against it, I will change it to `hasEntries` later.